cmd/shipper-app/main.go

```diff
@@ -348,19 +348,8 @@ func startInstallationController(cfg *cfg) (bool, error) {
 		return false, nil
 	}
 
-	dynamicClientBuilderFunc := func(gvk *schema.GroupVersionKind, config *rest.Config, cluster *shipper.Cluster) dynamic.Interface {
-		config.APIPath = dynamic.LegacyAPIPathResolverFunc(*gvk)
-		config.GroupVersion = &schema.GroupVersion{Group: gvk.Group, Version: gvk.Version}
-
-		if cfg.restTimeout != nil {
```

Comment:
Are we not copying the timeout anymore?

Reply:
We are, but now just once. The `restConfig` this `dynamicClientBuilderFunc` gets comes from the `clusterclientstore` (by means of its `GetConfig` method), and the timeout is already set here: https://github.com/bookingcom/shipper/blob/master/pkg/clusterclientstore/store.go#L304-L308